app/code/Magento/Checkout/view/frontend/web/js/sidebar.js

```diff
@@ -98,7 +98,9 @@ define([
 
                         /** @inheritdoc */
                         always: function (e) {
-                            e.stopImmediatePropagation();
+                            if (e) {
```

Comment:
Do we really need to check is event exists? It's framework lifecycle methods and the event should be always presented

Reply:
yes, we need to check is event exist cause as you see we are not passing any event in the `closeModal()` function and if we pass the event function then prompt will not close if we press the escape key cause in `prompt` component again we are not passing event in the `closeModal()`

Reply:
@konarshankar07 @VladimirZaets should we just remove the `always` function?

Reply:
@sivaschenko ... I don't think we can remove `always` function cause it depends on [here](https://github.com/magento/magento2/blob/c7ca62eaa2d5e6a8978dc9eeb00d14625c3ef893/app/code/Magento/Ui/view/base/web/js/modal/prompt.js#L169) but still not sure